### DISCUSSION TITLE
Chip selected color

### DISCUSSION THREAD

**Discussion by EliasDtz** [callstack/react-native-paper#3179] [Q&A]
I got a problem with the selectedColor prop of the chip component. As I understand it, it is supposed to change the color of the chip only when its selected but instead it changes the color of the border, icon and text. Am I using it wrong or missunderstanding its purpose?

![Unbenannt](https://user-images.githubusercontent.com/77106403/169362892-040b5cc9-5ef3-41bc-ba7d-8c29a483b160.PNG)
![Unbenannt2](https://user-images.githubusercontent.com/77106403/169362897-d03d6113-763e-486d-891b-429fd88c8c00.PNG)

**Comment by yaffalhakim1** [comment]
hey man, still didn't get the answer?

**Comment by theBGuy** [comment]
> I got a problem with the selectedColor prop of the chip component. As I understand it, it is supposed to change the color of the chip only when its selected but instead it changes the color of the border, icon and text. Am I using it wrong or missunderstanding its purpose?
> 
> ![Unbenannt](https://user-images.githubusercontent.com/77106403/169362892-040b5cc9-5ef3-41bc-ba7d-8c29a483b160.PNG)
> ![Unbenannt2](https://user-images.githubusercontent.com/77106403/169362897-d03d6113-763e-486d-891b-429fd88c8c00.PNG)
> 
> 

Wow. I came across this myself and opened a PR to fix it <https://github.com/callstack/react-native-paper/pull/4825>  but didn't realize it had been like this for long.
